app/models/concerns/cable_ready/updatable.rb

```diff
@@ -66,6 +74,36 @@ def enrich_association_with_updates(name, option)
           through_association = reflection.through_reflection.name.to_s.singularize
         end
 
+        options = build_options(option)
+
+        reflection.klass.send(:include, CableReady::Updatable) unless reflection.klass.respond_to?(:cable_ready_collections)
+
+        reflection.klass.cable_ready_collections.register({
+          klass: self,
+          foreign_key: reflection.foreign_key,
+          name: name,
+          inverse_association: inverse_of,
+          through_association: through_association,
+          options: options
+        })
+      end
+
+      def enrich_attachments_with_updates(name, option)
```

Comment:
This is awesome! The only caveat is that this class has grown a long beard up till now. I think there's an opportunity to factor out some of the `Association` / `ActiveStorage` related stuff here... let me ponder that a bit...